testsuite/features/secondary/min_bootstrap_negative.feature

```diff
@@ -67,6 +68,21 @@ Feature: Negative tests for bootstrap normal minions
      And I select the hostname of "proxy" from "proxies"
      And I click on "Bootstrap"
      And I wait until I see "Successfully bootstrapped host!" text
+     And I navigate to "rhn/systems/Overview.do" page
+     And I wait until I see the name of "sle_minion", refreshing the page
```

Comment:
Beware that sometimes this is not even enough, and you should wait for events like package list refresh to finish.